[2024-12-11T16:44:12.538+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-12-11T16:44:12.559+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dbt_orchestrator.dbt_connection_check.dbt_debug scheduled__2024-12-11T16:15:00+00:00 [queued]>
[2024-12-11T16:44:12.566+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dbt_orchestrator.dbt_connection_check.dbt_debug scheduled__2024-12-11T16:15:00+00:00 [queued]>
[2024-12-11T16:44:12.566+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-12-11T16:44:12.576+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): dbt_connection_check.dbt_debug> on 2024-12-11 16:15:00+00:00
[2024-12-11T16:44:12.581+0000] {standard_task_runner.py:63} INFO - Started process 311 to run task
[2024-12-11T16:44:12.584+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dbt_orchestrator', 'dbt_connection_check.dbt_debug', 'scheduled__2024-12-11T16:15:00+00:00', '--job-id', '42', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpx4ktq89d']
[2024-12-11T16:44:12.585+0000] {standard_task_runner.py:91} INFO - Job 42: Subtask dbt_connection_check.dbt_debug
[2024-12-11T16:44:12.625+0000] {task_command.py:426} INFO - Running <TaskInstance: dbt_orchestrator.dbt_connection_check.dbt_debug scheduled__2024-12-11T16:15:00+00:00 [running]> on host 49f8df2c9427
[2024-12-11T16:44:12.704+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Mahmoud' AIRFLOW_CTX_DAG_ID='dbt_orchestrator' AIRFLOW_CTX_TASK_ID='dbt_connection_check.dbt_debug' AIRFLOW_CTX_EXECUTION_DATE='2024-12-11T16:15:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-11T16:15:00+00:00'
[2024-12-11T16:44:12.705+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-12-11T16:44:12.716+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-12-11T16:44:12.716+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'dbt debug --profiles-dir /opt/***/dbt --project-dir /opt/***/dbt/ecommerce']
[2024-12-11T16:44:12.726+0000] {subprocess.py:86} INFO - Output:
[2024-12-11T16:44:14.759+0000] {subprocess.py:93} INFO - [0m16:44:14  Running with dbt=1.8.0
[2024-12-11T16:44:14.770+0000] {subprocess.py:93} INFO - [0m16:44:14  dbt version: 1.8.0
[2024-12-11T16:44:14.771+0000] {subprocess.py:93} INFO - [0m16:44:14  python version: 3.9.19
[2024-12-11T16:44:14.771+0000] {subprocess.py:93} INFO - [0m16:44:14  python path: /home/***/.local/bin/python
[2024-12-11T16:44:14.772+0000] {subprocess.py:93} INFO - [0m16:44:14  os info: Linux-6.8.0-49-generic-x86_64-with-glibc2.36
[2024-12-11T16:44:14.840+0000] {subprocess.py:93} INFO - [0m16:44:14  Using profiles dir at /opt/***/dbt
[2024-12-11T16:44:14.841+0000] {subprocess.py:93} INFO - [0m16:44:14  Using profiles.yml file at /opt/***/dbt/profiles.yml
[2024-12-11T16:44:14.842+0000] {subprocess.py:93} INFO - [0m16:44:14  Using dbt_project.yml file at /opt/***/dbt/ecommerce/dbt_project.yml
[2024-12-11T16:44:14.842+0000] {subprocess.py:93} INFO - [0m16:44:14  adapter type: postgres
[2024-12-11T16:44:14.843+0000] {subprocess.py:93} INFO - [0m16:44:14  adapter version: 1.8.0
[2024-12-11T16:44:14.919+0000] {subprocess.py:93} INFO - [0m16:44:14  [[33mWARNING[0m]: Deprecated functionality
[2024-12-11T16:44:14.919+0000] {subprocess.py:93} INFO - The `tests` config has been renamed to `data_tests`. Please see
[2024-12-11T16:44:14.919+0000] {subprocess.py:93} INFO - https://docs.getdbt.com/docs/build/data-tests#new-data_tests-syntax for more
[2024-12-11T16:44:14.920+0000] {subprocess.py:93} INFO - information.
[2024-12-11T16:44:14.963+0000] {subprocess.py:93} INFO - [0m16:44:14  Configuration:
[2024-12-11T16:44:14.964+0000] {subprocess.py:93} INFO - [0m16:44:14    profiles.yml file [[32mOK found and valid[0m]
[2024-12-11T16:44:14.965+0000] {subprocess.py:93} INFO - [0m16:44:14    dbt_project.yml file [[32mOK found and valid[0m]
[2024-12-11T16:44:14.965+0000] {subprocess.py:93} INFO - [0m16:44:14  Required dependencies:
[2024-12-11T16:44:14.975+0000] {subprocess.py:93} INFO - [0m16:44:14   - git [[32mOK found[0m]
[2024-12-11T16:44:14.976+0000] {subprocess.py:93} INFO - 
[2024-12-11T16:44:14.976+0000] {subprocess.py:93} INFO - [0m16:44:14  Connection:
[2024-12-11T16:44:14.976+0000] {subprocess.py:93} INFO - [0m16:44:14    host: 172.18.0.3
[2024-12-11T16:44:14.977+0000] {subprocess.py:93} INFO - [0m16:44:14    port: 5432
[2024-12-11T16:44:14.977+0000] {subprocess.py:93} INFO - [0m16:44:14    user: ***
[2024-12-11T16:44:14.978+0000] {subprocess.py:93} INFO - [0m16:44:14    database: ecommerce
[2024-12-11T16:44:14.978+0000] {subprocess.py:93} INFO - [0m16:44:14    schema: oltp
[2024-12-11T16:44:14.979+0000] {subprocess.py:93} INFO - [0m16:44:14    connect_timeout: 10
[2024-12-11T16:44:14.979+0000] {subprocess.py:93} INFO - [0m16:44:14    role: None
[2024-12-11T16:44:14.980+0000] {subprocess.py:93} INFO - [0m16:44:14    search_path: None
[2024-12-11T16:44:14.980+0000] {subprocess.py:93} INFO - [0m16:44:14    keepalives_idle: 0
[2024-12-11T16:44:14.981+0000] {subprocess.py:93} INFO - [0m16:44:14    sslmode: None
[2024-12-11T16:44:14.981+0000] {subprocess.py:93} INFO - [0m16:44:14    sslcert: None
[2024-12-11T16:44:14.982+0000] {subprocess.py:93} INFO - [0m16:44:14    sslkey: None
[2024-12-11T16:44:14.983+0000] {subprocess.py:93} INFO - [0m16:44:14    sslrootcert: None
[2024-12-11T16:44:14.983+0000] {subprocess.py:93} INFO - [0m16:44:14    application_name: dbt
[2024-12-11T16:44:14.983+0000] {subprocess.py:93} INFO - [0m16:44:14    retries: 1
[2024-12-11T16:44:14.984+0000] {subprocess.py:93} INFO - [0m16:44:14  Registered adapter: postgres=1.8.0
[2024-12-11T16:44:15.151+0000] {subprocess.py:93} INFO - [0m16:44:15    Connection test: [[32mOK connection ok[0m]
[2024-12-11T16:44:15.151+0000] {subprocess.py:93} INFO - 
[2024-12-11T16:44:15.151+0000] {subprocess.py:93} INFO - [0m16:44:15  [32mAll checks passed![0m
[2024-12-11T16:44:15.404+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-12-11T16:44:15.405+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-12-11T16:44:15.432+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dbt_orchestrator, task_id=dbt_connection_check.dbt_debug, run_id=scheduled__2024-12-11T16:15:00+00:00, execution_date=20241211T161500, start_date=20241211T164412, end_date=20241211T164415
[2024-12-11T16:44:15.483+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-12-11T16:44:15.521+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-11T16:44:15.523+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
